README.md

```diff
@@ -1,6 +1,7 @@
 # Journeys School 6th Grade Programming
 
 Goals:
+
```

Comment:
Above ^^ you will see a green line with "+".  That indicates that I added a blank line to the file.  This view is called a "diff" view to show differences between my file and the file in master.

Diffs will have green lines with a "+" showing lines added, and red lines with a "-" showing lines removed.